css/wishlist.css

```diff
@@ -0,0 +1,46 @@
+.main-container{
+    margin: 8vh 1rem 2rem 1rem;
```

Comment:
8vh 1rem 2rem would work in similar manner
[When three values are specified, the first value applies to the top, the second to the right and left, the third to the bottom.]

Reply:
oh yes, will change it.